CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,30 @@
+# fre-cli Code of Conduct
+
+## Purpose
+Modeling Systems Division, and GFDL as whole, seeks to create a friendly, safe and welcoming environment for all, regardless of gender, sexual orientation, ability, ethnicity, socioeconomic status, and religion. In order to protect this collaborative environment, the following Code of Conduct sets out a series of guidelines to follow while interacting with the fre-cli repository, and its contributors. This code applies equally to mentors, mentees, and independent contributors.
+
+## Expected Behavior
+* Demonstrating empathy and kindness toward other people
+* Being respectful of differing opinions, viewpoints, and experiences
+* Giving and gracefully accepting constructive feedback
+* Accepting responsibility and apologizing to those affected by our mistakes,
+  and learning from the experience
+* Focusing on what is best not just for us as individuals, but for the overall
+  community
+
+## Unacceptable Behavior
+* The use of sexualized language or imagery, and sexual attention or advances of
+  any kind
+* Trolling, insulting or derogatory comments, and personal or political attacks
+* Public or private harassment
+* Publishing others' private information, such as a physical or email address,
+  without their explicit permission
+* Other conduct which could reasonably be considered inappropriate in a
+  professional setting
+
+## Addressing Grievances:
+Unacceptable behavior from any community member will not be tolerated. Anyone asked to stop unacceptable behavior is expected to comply immediately. Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by emailing gfdl.climate.model.info@noaa.gov 
+
+## Attribution
+This Code of Conduct is adapted from the [Contributor Covenant], version 1.4,
+available at https://www.contributor-covenant.org/version/1/4/code-of-conduct.html, the [Citizen Code of Conduct] available at https://web.archive.org/web/20200330154000/http://citizencodeofconduct.org/, and the [GFDL Code of Conduct] available at https://www.gfdl.noaa.gov/wp-content/uploads/2022/05/GFDL_Code_of_Conduct.pdf
```

Comment:
are `[Contributor Covenant]`, `[Citizen Code of Conduct]`, and `[GFDL Code of Conduct]` supposed to be hyperlinks that point you to the links?

To make it clickable then, one would do, e.g. `[Contributor Covenant](Contributor Covenant url)`, like this: [Contributor Covenant](https://www.contributor-covenant.org/version/1/4/code-of-conduct.html)

Reply:
The links all work now, but it's odd that the Citizen Code of Conduct is only available on the Internet Archive. Is the Citizen Code of Conduct available somewhere else?